source/atlas-cli-deploy-local.txt

```diff
@@ -74,7 +74,14 @@ Supported OS for Local Atlas Deployments
 Complete the Prerequisites
 --------------------------
 
-.. include:: /includes/fact-deploy-tutorial-prereqs.rst
+Before you begin, complete the following prerequisites:
+
+.. important::
+
+   For compatibility information for each prerequisite product, see the 
```

Comment:
Suggestion: 

For compatibility information on each product in the dependencies list, see the product's installation documentation.